docs/AccordionForm.md

```diff
@@ -261,6 +261,8 @@ For the previous example, the data sent to the `dataProvider` will be:
 
 **Note:** Setting the `sanitizeEmptyValues` prop to `true` will also have a (minor) impact on react-admin inputs (like `<TextInput>`, `<NumberInput>`, etc.): empty values (i.e. values equal to `null`) will be removed from the form state on submit, unless the record actually had a value for that field.
 
+**Note** Even with `sanitizeEmptyValues` set to `true`, deeply nested fields won't be set to `null` nor removed. If you need to sanitize those fields, use the [the `transform` prop](./Edit.md#transform) of `<Edit>` or `<Create>` components.
```

Comment:
I used to be a big fan of [The The](https://en.wikipedia.org/wiki/The_The) (the band), but I don't think they fit in here.